src/test.rs

```diff
@@ -77,6 +79,7 @@ fn test_zero_cap() {
     let b = arena.alloc(2);
     assert_eq!(*a, 1);
     assert_eq!(*b, 2);
+    assert_eq!(arena.len(), 2);
```

Comment:
Can you also add a test that exercises `len` when the arena has multiple chunks? Thanks.

Reply:
I believe I do this in the test `arena_as_intended`. Is this enough or is more needed?

Reply:
Ah yeah I missed that that test was exercising that path. Just the other debug assertion and then we can merge this, then.